sphinx/user-guide/installation.rst

```diff
@@ -0,0 +1,427 @@
+.. include:: ../hyperlinks.rst
+   :start-line: 1
+
+Installation
+============
+
+.. _archives: https://github.com/metomi/rose/tags
+
+The source code for Rose is available via `github <rose github>`_, code releases
+are available in ``zip`` and ``tar.gz`` `archives`_.
+
+1. Un-pack the archive file into an appropriate location on your system.
+2. Add the ``rose/bin/`` directory into your ``PATH`` environment variable.
+3. Check system compatability by running ``rose check-software``.
+
+
+System Requirements
+-------------------
+
+Rose runs on Unix/Linux systems and is known to run on RHEL6 and a number of
+systems including those documented under `metomi-vms`_.
+
+System compatibility can be tested using the ``rose check-software`` command.
+
+.. script-include:: rose check-software --rst
```

Comment:
See my comments on `bin/rose-check-software` regarding auto-loading of the spec here.